lib/validations.js

```diff
@@ -453,6 +456,14 @@ Validatable.prototype.isValid = function (callback, data) {
       });
     });
 
+    if (reportDiscardedProperties) {
+      for (var ix in inst.__discardedProperties) {
+        var key = inst.__discardedProperties[ix];
+        inst.errors.add(key, defaultMessages['absence'], 'absence');
```

Comment:
To me `unknown` means "unknown error", not "unknown property". I have reworked the code to `unknown-property`.